protocols/v1/src/tempCodeRunnerFile.rs

```diff
@@ -0,0 +1 @@
+Arbitrary
```

Comment:
What is this file for?

Reply:
It was there in the tempCodeRunnerFile.rs before, I thought it'd not be a good idea to touch it, since i just started rust.